src/WinUI.TableView.csproj

```diff
@@ -1,13 +1,27 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Uno.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
-    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
-    <WindowsSdkPackageVersion>10.0.22621.38</WindowsSdkPackageVersion>
+    <TargetFrameworks>
+      net8.0-desktop;
+      net8.0;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.26100;
+      net8.0-browserwasm;
```

Comment:
Noticed we aren't including android/ios tfms 

It may work nicely on mobile as well under certain circumstances. Could be interesting to add them and see how it behaves in the sample app

Reply:
I did that intentionally cause TableView is not made for small screens. I tried it on android and that wasn't good. I'll try to make it available for small screens in 1.4 hopefully.